index.bs

```diff
@@ -795,6 +743,562 @@ the following list:
 
 </div>
 
+
+
+# The {{WebSocketStream}} interface # {#the-websocketstream-interface}
+
+The Web IDL definition for the {{WebSocketStream}} class is given as follows:
+
+<xmp class="idl">
+dictionary WebSocketOpenInfo {
+  ReadableStream readable;
+  WritableStream writable;
+  DOMString extensions;
+  DOMString protocol;
+};
+
+dictionary WebSocketCloseInfo {
+  [Clamp] unsigned short closeCode;
+  USVString reason = "";
+};
+
+dictionary WebSocketStreamOptions {
+  sequence<USVString> protocols;
+  AbortSignal signal;
+};
+
+[
+    Exposed=(Window,Worker)
+] interface WebSocketStream {
```

Comment:
Nit: this would look neater and be more consistent as

```webidl
[Exposed=(Window,Worker)]
interface WebSocketStream {
```

Reply:
Done.

Reply:
Looks like I only changed WebSocketError. Sorry. Actually properly really changed now.